src/diagram.ts

```diff
@@ -238,6 +240,31 @@ export class Diagram {
         }
     }
 
+    undisplayNodes(): void {
```

Comment:
I'd think that "hide" and "show" sound more natural instead of "display" and "undisplay".

Reply:
Good call. Done.